sklearn/multiclass.py

```diff
@@ -144,10 +144,10 @@ class OneVsRestClassifier(BaseEstimator, ClassifierMixin, MetaEstimatorMixin):
 
     This strategy can also be used for multilabel learning, where a classifier
     is used to predict multiple labels for instance, by fitting on a 2-d matrix
-    in which cell [i, j] is 1 if sample i has label j and 0 otherwise.
-
-    In the multilabel learning literature, OvR is also known as the binary
-    relevance method.
+    in which cell [i, j] is 1 if sample i has label j and 0 otherwise. In this
+    case, one classifier is trained for each label independent of the other
+    labels. In the multilabel learning literature, OvR is also known as the
```

Comment:
I don't think it's accurate to say this is OvR. I think it's fair to even call this "A related strategy" above, and change "OvR is also" to "this is".